youtube_dl/extractor/myspass.py

```diff
@@ -35,11 +35,14 @@ def _real_extract(self, url):
         title = xpath_text(metadata, 'title', fatal=True)
         video_url = xpath_text(metadata, 'url_flv', 'download url', True)
         video_id_int = int(video_id)
-        for group in re.search(r'/myspass2009/\d+/(\d+)/(\d+)/(\d+)/', video_url).groups():
-            group_int = int(group)
-            if group_int > video_id_int:
-                video_url = video_url.replace(
-                    group, compat_str(group_int // video_id_int))
+
+        regex_check = re.search(r'/myspass2009/\d+/(\d+)/(\d+)/(\d+)/', video_url)
```

Comment:
Although this is fine logically, this change is a bit shorter without changing the indentation of following lines.
```
        grps = re.search(r'/myspass2009/\d+/(\d+)/(\d+)/(\d+)/', video_url)
        for group in grps.groups() if grps else []:
```

Reply:
#30676